symphony/bdk/core/config/model/bdk_config.py

```diff
@@ -26,6 +26,12 @@ def __init__(self, **config):
         self.app = BdkAppConfig(config.get("app"))
         self.datafeed = BdkDatafeedConfig(config.get("datafeed"))
 
+    def setBotConfig(self, private_key_content=None, certificate_content=None):
+        if private_key_content is not  None :
+            self.bot.setPrivatekey(path=None, private_key_content=private_key_content)
```

Comment:
do we agree that the path will always be None as the content is given by string ?